community/kernel/src/main/java/org/neo4j/kernel/internal/GraphDatabaseAPI.java

```diff
@@ -48,6 +52,43 @@
      */
     URL validateURLAccess( URL url ) throws URLAccessValidationError;
 
-    @Deprecated
     String getStoreDir();
+
+    /**
+     * Returns all relationship types currently in the underlying store.
+     * Relationship types are added to the underlying store the first time they
+     * are used in a successfully commited {@link Node#createRelationshipTo
+     * node.createRelationshipTo(...)}. Note that this method is guaranteed to
+     * return all known relationship types, but it does not guarantee that it
+     * won't return <i>more</i> than that (e.g. it can return "historic"
+     * relationship types that no longer have any relationships in the node
+     * space).
+     *
+     * @return all relationship types in the underlying store
+     */
+    ResourceIterable<RelationshipType> getAllRelationshipTypesInUseAndNot();
```

Comment:
@lutovich Oh wow, I was completely wrong. I followed the path down to call to `inUse(...)` which apparently is overloaded as a term and means (2). Mmh, looks like we've changed the behaviour of `getAllXX` methods now, which might not be a good idea.

Can't we just move over both `getAllXX` and `getAllXXInUse` (neither of these returning !inUse records i.e. !AbstractBaseRecord#inUse()), just like GGO had? That would keep current behaviour and we wouldn't need to invent new names.